### ISSUE TITLE
Error while bootstraping Snuba on Rocky 9

### ISSUE THREAD

**Issue by remi-fongaufier** [getsentry/self-hosted#1828]
### Self-Hosted Version

22.11.0

### CPU Architecture

x86_64

### Docker Version

20.10.21

### Docker Compose Version

2.12.2

### Steps to Reproduce

1. Execute the script `install.sh`

### Expected Result

Installation should be succesful

### Actual Result

An error occurs when installation try to bootstrap `Snuba`:
▶ Bootstrapping and migrating Snuba ...
Network sentry-self-hosted_default  Creating
Network sentry-self-hosted_default  Created
Container sentry-self-hosted-redis-1  Creating
Container sentry-self-hosted-clickhouse-1  Creating
Container sentry-self-hosted-zookeeper-1  Creating
Container sentry-self-hosted-redis-1  Created
Container sentry-self-hosted-zookeeper-1  Created
Container sentry-self-hosted-kafka-1  Creating
Container sentry-self-hosted-clickhouse-1  Created
Container sentry-self-hosted-kafka-1  Created
Container sentry-self-hosted-redis-1  Starting
Container sentry-self-hosted-clickhouse-1  Starting
Container sentry-self-hosted-zookeeper-1  Starting
Container sentry-self-hosted-redis-1  Started
Container sentry-self-hosted-zookeeper-1  Started
Container sentry-self-hosted-zookeeper-1  Waiting
Container sentry-self-hosted-clickhouse-1  Started
container for service "zookeeper" is unhealthy
Error in bootstrap-snuba.sh:3.
'$dcr snuba-api bootstrap --no-migrate --force' exited with status 1
-> ./install.sh:main:31
--> bootstrap-snuba.sh:source:3

**Comment by chukaif** [comment]
I encountered the same problem when trying to install 'self-hosted-22.11.0'

**Comment by ethanhs** [comment]
We have found that sometimes zookeeper is unhealthy intermittently. If you try again, does it work?

**Comment by chukaif** [comment]
> We have found that sometimes zookeeper is unhealthy intermittently. If you try again, does it work?

no, it does't work, I tried several times. i also tried 'self-hosted-22.10.0', it does't work too. my server use 'CentOS Stream 9'

**Comment by w-ap-admin** [comment]
I try to install Sentry Self-hosted several times and it never works.
I don't know if it will help you, but the installation works with Rocky 8.

**Comment by chukaif** [comment]
i change the server OS from 'CentOS Stream 9' to 'Ubuntu Server 22.04 LTS', then it works.

**Comment by Nopsty** [comment]
Same issue on Alma 9
I also tried newer 5.x versions of confluentinc/cp-zookeeper:5.x.x

**Comment by ethanhs** [comment]
Since it seems that a number of people are hitting this, I've added it to our backlog to investigate further.

**Comment by farfromrefug** [comment]
@ethanhs i am stuck on that error too. Tried every single solution from the issues and nothing worked. No more sentry here :s

**Comment by gitbugr** [comment]
Same issue here, also on Alma 9

**Comment by aminvakil** [comment]
This may be related to the issue reported here: https://github.com/getsentry/self-hosted/issues/1445#issuecomment-1122415883

Can you try this workaround and report back?

**Comment by bobohb** [comment]
> This may be related to the issue reported here: [#1445 (comment)](https://github.com/getsentry/self-hosted/issues/1445#issuecomment-1122415883)
> 
> Can you try this workaround and report back?

sloved

test environment：
AlmaLinux release 9.1
docker-ce-3:20.10.22-3.el9.*
docker-ce-cli-1:20.10.22-3.el9.*
docker-ce-rootless-extras-0:20.10.22-3.el9.*
docker-compose-plugin-0:2.14.1-3.el9.*
containerd.io-0:1.6.13-3.1.el9.*

**Comment by dereckson** [comment]
If it's RHEL specific, it's perhaps worthwhile to retry it in SELinux permissive mode: `setenforce Permissive`, then check audit log for failures if it works there
